src/main/java/com/minecolonies/colony/buildings/BuildingMiner.java

```diff
@@ -54,10 +105,21 @@
      */
     public                  boolean             clearedShaft            = false;
 
-    //Save in hut
+    /**
+     * The starting level of the node.
+     */
     private int startingLevelNode = 0;
+    /**
+     * The id of the active node.
+     */
     private int active = 0;
+    /**
+     * The number of the current level.
+     */
     private int currentLevel = 0;
+    /**
+     * The position of the start of the shaft.
+     */
     private BlockPos shaftStart;
 
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)